static/specification.html

```diff
@@ -704,6 +704,22 @@ <h3>
           </section>
         </section>
       </section>
+      <section id="permissions-policy" data-cite="permissions-policy">
+        <h2>Permissions Policy integration</h2>
+        <p>
+          This specification defines a [=policy-controlled feature=] identified
+          by the string <dfn class="permission">"monetization"</dfn>". Its
+          <a>default allowlist</a> is `'self'`.
+        </p>
+        <aside class="note">
+          <p>
+            A <a>document</a>’s [=Document/permissions policy=] determines
+            whether [^link^] elements are monetized - particularly in
+            third-party content, such as ifr. When disabled, monetization is
```

Comment:
"such as ifr" -> "such as in iframes" ? 
It's obvious what you mean, but ??

Reply:
oops! good catch.